examples/transportation/segment/road/destinations/road-destinations.yaml

```diff
@@ -0,0 +1,34 @@
+---
+id: overture:transportation:segment:example:destinations:1
+type: Feature
+geometry:
+  type: LineString
+  coordinates: [[0, 0], [1, 1]]
+properties:
+  theme: transportation
+  type: segment
+  subtype: road
+  class: secondary
+  version: 0
+  update_time: "2024-04-24T00:00:00-00:00"
+  connector_ids: [overture:transportation:connector:123, overture:transportation:connector:678]
+  destinations:
+    - labels:
+      - value: Seattle
+        type: place
+      - value: Main Street
+        type: street
+      - value: I90
+        type: route
+      symbols:
+      - motorway
+      - airport
+      from:
+      - segment_id: overture:transportation:segment:234
+        connector_id: overture:transportation:connector:234
+      - segment_id: overture:transportation:segment:567
+        connector_id: overture:transportation:connector:567
+        when:
+          heading: forward
```

Comment:
Interesting. Isn't travel direction already implied by the sequence?

I think `when` is only allowed here because we mistakenly forgot to set `unevaluatedProperties: false` in the sequence item model.

Reply:
New comment, during convo. w. Cozmin, not entirely related but with clearer understanding now.

Current segment, which is the "to" in the from-to model, would have:

```
when: { heading: backward }
```

On the from segments in the `from` list, would have just an `initial_heading` property.

That's the way that would make it most consistent w prohibited transitions without flipping the from-to direction.

----

If we flipped it so that destinations live on the from, then it would be:

```
when: { heading: forward }
```

And the `from` list would become `to` and that would have `final_heading: backward`.

Reply:
as discussed, when.heading and final_heading kept after moving this property on the source/from segment